src/net_processing.cpp

```diff
@@ -779,6 +779,21 @@ void PeerLogicValidation::InitializeNode(CNode *pnode) {
         PushNodeVersion(pnode, connman, GetTime());
 }
 
+// Note: unbroadcast transactions will be filtered through filterInventoryKnown
+void PeerLogicValidation::ReattemptInitialBroadcast(CScheduler& scheduler) const
+{
+    std::set<uint256> unbroadcast_txids;
+    WITH_LOCK(m_mempool.cs, unbroadcast_txids = m_mempool.m_unbroadcast_txids);
+
+    for (const uint256& txid : unbroadcast_txids) {
+        RelayTransaction(txid, *connman);
+    }
+
+    // schedule next run for 10-15 minutes in the future
+    const std::chrono::milliseconds delta = std::chrono::milliseconds(10 * 60 * 1000 + GetRand(5 * 60 * 1000));
```

Comment:
nit: could declare these two times as constants.

Reply:
pass on this one, I pulled `delta` out for readability, but breaking into two constants seems a bit overkill to me